includes/SiteGen/SiteGen.php

```diff
@@ -56,9 +56,10 @@ class SiteGen {
 	private static function check_capabilities() {
 		$capability = new SiteCapabilities();
 
-		$ai_enabled = $capability->get( 'canAccessAI' );
+		$sitegen_enabled = $capability->get( 'hasAISiteGen' );
+		$ai_enabled      = $capability->get( 'canAccessAI' );
```

Comment:
@arunshenoy99 I think @wpscholar had mentioned wanting to retire canAccessAI in favor of feature specific capabilities.

Reply:
yeah, I think the hasAISitegen would be the only capability we will need here.

Reply:
@wpscholar can you confirm whether it's safe to check only the `hasAISiteGen` capability?